src/android/MsalPlugin.java

```diff
@@ -560,7 +560,19 @@ private JSONArray processClaims(Map<String, ?> claims) {
             try {
                 JSONObject claimObj = new JSONObject();
                 claimObj.put("key", claim.getKey());
-                claimObj.put("value", claim.getValue());
+                if (claim.getValue() instanceof net.minidev.json.JSONArray) {
+                  claimObj.put("value", new JSONArray(claim.getValue().toString()));
+                }
```

Comment:
Let's do `} else if (...) {` and `} else {` to match the existing brace layout in this file I think